core/src/main/java/org/radargun/stages/lifecycle/ServiceStartStage.java

```diff
@@ -53,67 +70,84 @@
    @InjectTrait
    private ConfigurationProvider configurationProvider;
 
+   @Property(doc = "Start only the given slave index")
+   private int slaveIndex = -1;
```

Comment:
So it seems like this PR is providing two options to achieve this goal? Either using `StaggerSlaveStartupModeEnum`=`ASYNC` or starting specific nodes by index? Is that correct?